docs/src/solvers/ode_solve.md

```diff
@@ -95,6 +95,22 @@ parabolic PDEs, and thus it's suggested you replace it with `Rodas4P` in those
 situations which is 3rd order. `ROS3P` is only third order and achieves 3rd order
 on such problems and can thus be more efficient in this case.
 
+#### Exponential Integrators for Stiff Problems
+
+If explicit methods are preferred, then the exponential integrators provide an alternative
```

Comment:
This section doesn't seem to be a high level description of the cases where it's useful and benchmarks well vs the other algorithms